open_diloco/train_fsdp.py

```diff
@@ -218,7 +212,7 @@ def train(config: Config):
 
     # DataLoader preparation
     tokenizer = AutoTokenizer.from_pretrained("mistralai/Mistral-7B-v0.1", use_fast=True)
-    tokenizer.pad_token = "</s>"  # Ensure pad token is set for models that need it
+    tokenizer.pad_token = " { "  # Ensure pad token is set for models that need it
```

Comment:
cursor strikes again!

Reply:
this one is on me: git rebase skill issue :cry: